src/ucs/sys/topo/base/topo.c

```diff
@@ -443,8 +445,6 @@ ucs_topo_get_distance_sysfs(ucs_sys_device_t device1,
     }
 
     /* Report best perf for common PCI bridge or sysfs parsing error */
-default_distance:
-    status = ucs_topo_get_distance_default(device1, device2, distance);
```

Comment:
by removing this line, the "else" case of the if in line 433 behavior will change and may leave the distance not initialized

Reply:
re-added the line at the end